webcompat/static/js/lib/models/issue.js

```diff
@@ -91,13 +91,11 @@ issues.Issue = Backbone.Model.extend({
 
   getDescription: function(body) {
     // Get the Description of the body
-    var el = body;
-    el = $("strong:contains('Description')", el);
-    if (el[0]) {
-      var description = el[0].nextSibling.textContent;
-      description = description.replace(": ", "");
-      return description != null ? description.slice(0, 74) : null;
-    }
+    var regex = /<strong>Description<\/strong>:(.*)+\n<strong>Steps to Reproduce<\/strong>:/gm;
```

Comment:
Ohh, good point about the old issues.
Let me check.

Reply:
Yeah.. It just breaks and doesn't render anything because of the `el[1]` thing.

This PR was made so Karl could check the code I was writing. I wasn't intending to merge it. Sorry for not making it clear. I'll keep your ideas in my, though.
Thanks for the review @miketaylr :)

Reply:
I was thinking... Probably we'll have to write an `if` to treat this old issue case?

Reply:
Check the old code with blame. I was handling the old issues case. :)

Reply:
Ok!